src/layout/layout.rs

```diff
@@ -37,7 +37,7 @@ type Cache = LruCache<(Rect, Layout), (Segments, Spacers)>;
 const FLOAT_PRECISION_MULTIPLIER: f64 = 100.0;
 
 thread_local! {
-    static LAYOUT_CACHE: OnceLock<RefCell<Cache>> = const { OnceLock::new() };
+    static LAYOUT_CACHE: RefCell<Cache> = RefCell::new(Cache::new(NonZeroUsize::new(Layout::DEFAULT_CACHE_SIZE).unwrap()));
```

Comment:
This is strange that this didn't get caught by the format lint as too wide. Maybe due to being in a macro?

Reply:
Yeah, fmt never seems to work right in macros.